hyperhtml/documentation/index.html

```diff
@@ -180,14 +179,12 @@ <h3 id="introduction-1">Under the hood</h3>
               );
               </code></pre>
               <p>
-                This native feature, well reflected even once transpiled via Babel,
-                is the key to generate a DOM structure and parse it once,
-                addressing all the internal operations needed to update interpolations,
-                creating a unique and convenient relationship between a template literal and a DOM template.
+                This native feature is key to generating a DOM structure and parsing it once, 
```

Comment:
cleaner sentence, but the fact that Babel transpiled code preserves this behavior got lost in the PR.

Do you think it's irrelevant as detail for this kind of documentation?

I'm just really paranoid when it comes to transpilers because of the non standard mess they could produce in some cases (builtin classes extends to name one).